src/main/java/org/generationcp/middleware/api/brapi/ObservationServiceBrapi.java

```diff
@@ -0,0 +1,16 @@
+package org.generationcp.middleware.api.brapi;
+
+import org.generationcp.middleware.api.brapi.v2.observation.ObservationDto;
+import org.generationcp.middleware.api.brapi.v2.observation.ObservationSearchRequestDto;
+
+import java.util.List;
+
+public interface ObservationServiceBrapi {
+
+	List<ObservationDto> searchObservations(ObservationSearchRequestDto observationSearchRequestDto, Integer pageSize,
+		Integer pageNumber);
+
+	long countObservations(ObservationSearchRequestDto observationSearchRequestDto);
+
+	List<Integer> createObservations(List<ObservationDto> observations);
```

Comment:
We should just return List<ObservationDto> to avoid querying again in BMSAPI service. Also let's create an `ObservationImportRequest` class to follow the same pattern as other brapi services.
`List<ObservationDto> createObservations(List<ObservationImportRequest> observationImportRequestList);`